tests/src/test/scala/com/nvidia/spark/rapids/ParseDateTimeSuite.scala

```diff
@@ -228,11 +228,8 @@ class ParseDateTimeSuite extends SparkQueryCompareTestSuite with BeforeAndAfterE
           .repartition(2)
           .withColumn("c1", unix_timestamp(col("c0"), "yyyy-MM-dd HH:mm:ss"))
     }
-    val startTimeSeconds = System.currentTimeMillis()/1000L
     val cpuNowSeconds = withCpuSparkSession(now).collect().head.toSeq(1).asInstanceOf[Long]
     val gpuNowSeconds = withGpuSparkSession(now).collect().head.toSeq(1).asInstanceOf[Long]
-    assert(cpuNowSeconds >= startTimeSeconds)
```

Comment:
What is the reason for removing these assertions? Are they no longer valid?

Reply:
Yes, under spark 3.2+, the result will be zero instead of current time, since `NOW` is not longer being parsed in 3.2.

Reply:
Then have the test check the version. and have it check that now is replaced with a 0 instead.  Or just have the test skip entirely if it is 3.2+